services/log-management.html.md.erb

```diff
@@ -58,12 +60,18 @@ To set up a communication channel between the log management service and your Cl
 
 You can create a syslog drain service and bind apps to it using Cloud Foundry Command Line Interface (cf CLI) commands.
 
-1. To create the service instance, run `cf create-user-provided-service` (or `cf cups`) with the `-l` flag, filling in values as follows:
+1. To create the service instance, run `cf create-user-provided-service` (or `cf cups`) with the `-l`.
   - DRAIN-NAME: A name to use for your syslog drain service instance.
   - SYSLOG-DRAIN-URL: The syslog URL from [Step 1: Configure the Log Management Service](#step1).
 <pre class="terminal">
 $ cf create-user-provided-service DRAIN-NAME -l SYSLOG-URL
 </pre>
+
```

Comment:
I assume if this empty line is removed the rendered content would be on the same indentation level.

Reply:
Done